source/storage/apis-1.5.rst

```diff
@@ -338,26 +380,60 @@ collection.
     Posted BSOs whose ids do not appear in either "success" or "failed"
     should be treated as having failed for an unspecified reason.
 
-    Two input formats are available for multiple record POST requests,
-    selected by the *Content-Type* header of the request:
+    To allow upload of large numbers of items while ensuring that other
+    clients do not sync down inconsistent data, this API can combine several
+    POST requests into a single "batch" so that all modified BSOs appear to
+    have been submitted at the same time.  Batching behaviour is controlled
+    by the following query parameters:
 
-    - **application/json**: the input is a JSON list of objects, one for
-      for each BSO in the request.
+    - **batch**: indicates that uploads should be batched together into a
+      single conceptual update.  To begin a new batch pass the string 'true'.
+      To add more items to an existing batch pass a previously-obtained batch
+      identifier.
 
-    - **application/newlines**: each BSO is sent as a separate JSON object
-      followed by a newline.
+    - **commit**: indicates that the batch should be committed, and all items
+      uploaded to that batch made visible to other clients.  If present, it
+      must be the string 'true' and be combined with a batch identifier.
```

Comment:
very minor, but the "be combined with" part seems a bit awkward - I'd suggest something like "and a batch query parameter must also be specified with the batch identifier"